Add files via upload

IzadoraT opened the pull request:
Parte 4 - Etapa 1 
Caso de uso e diagramas de sequência

IzadoraT committed:
Add files via upload

Parte 4 - Etapa 1 
Caso de uso e diagramas de sequência

RodrigoTogisaki commented:
Parte 4 - Etapa 1 ----> Diagrama de classes

![Seção 04 - Diagrama de Classe UML](https://github.com/poo-ee-2024-1/g3/assets/167814493/f3c9a22f-aaf2-42c6-b7b8-f3b13ca7670a)

RodrigoTogisaki commented:
Parte 3 - Etapa 1  -----> Divisão de tarefas

Aqui neste File, será realizado a divisão, podendo este mudar a depender do tempo, para separação em partes para cada integrante de acordo com a parte que estará sendo responsabilizado
para concluir o trabalho em grupo e também o projeto propriamente.

Rodrigo - Líder: Gestão de tarefas, revisão e Back-end
Izadora: Back-end --> Modelos Abstratos e Interfaces
Eduardo: Back-end --> Modelos concretos e Controller
Gabriel Monteiro: Front-end --> Interface gráfica e view
Mauro Dalcin: Front-end --> Interface gráfica e view

Assim estará definido de maneira, a não ser necessariamente permanente, para caso necessite ter uma maior flexibilidade para ajudar em outras fases que estiver sendo necessário, para tal será realizado, quando possível, uma conversa semanal para todos terem noção do progresso feito, para caso seja necessário realocar ajuda.

gabrielmonteirog commented:
Parte 2: Plano - Objetivo geral
https://docs.google.com/document/d/19_XmMaW8dszwCdjOl9Q6bAMGMW1HLpgJMcLSX_hrSgo/edit?usp=sharing

mredurl1 commented:
Parte 1: Introdução
[poo seção 1.docx](https://github.com/user-attachments/files/16104235/poo.secao.1.docx)

Maurodalcin commented:
Marketplace Virtual
https://docs.google.com/document/d/19_XmMaW8dszwCdjOl9Q6bAMGMW1HLpgJMcLSX_hrSgo/edit?usp=sharing

IzadoraT committed:
main

front-end

nem todos os arquivos serão usados, alguns são referentes a funções ou campos a que seriam implementadas nas próximas versões

IzadoraT committed:
main

main

IzadoraT committed:
main

main

IzadoraT committed:
main

main
